os/platformcommondrmdefines.cpp

```diff
@@ -79,6 +79,10 @@ int CreateFrameBuffer(
     ret = drmModeAddFB2WithModifiers(
         gpu_fd, iwidth, iheight, iframe_buffer_format, m_igem_handles,
         m_ipitches, m_ioffsets, modifiers, fb_id, DRM_MODE_FB_MODIFIERS);
+
+    if (ret == 0) {
```

Comment:
there are a few modificators that is not related with RBC as I know.
For example, Yf tile.
We need to check exactly the CCS modificator I think.

Reply:
IIRC Y-tiling is a requirement for RBC though so it is required and RBC will fail without it. What's the current set of modifiers in use?

Reply:
both I915_FORMAT_MOD_Y_TILED_CCS and I915_FORMAT_MOD_Yf_TILED_CCS modifier are possible. In current impl, when the plane initialize, property will be queried what modifier is supported and set it to be prefer modifier, with which later to allocate offscreen render target and frame buffer. If allocate fails, will add it into blacklist. so I think we dont have to worry about this.

Reply:
I915_FORMAT_MOD_Yf_TILED how about this one

Reply:
Only I915_FORMAT_MOD_Y_TILED_CCS and I915_FORMAT_MOD_Yf_TILED_CCS is applicable for RBC, for primary plane. I915_FORMAT_MOD_Yf_TILED is not taking account.